client/src/components/Table.js

```diff
@@ -1,12 +1,9 @@
 import React from "react";
```

Comment:
@tesla809 I just remimbered that you asked e to use
`import React, {Component} from 'react`
I can do that but may I ask why? what is the difference?